content/accessible/labels-and-text-alternatives/index.md

```diff
@@ -0,0 +1,334 @@
+---
+page_type: guide
+title: Labels and text alternatives
+author: robdodson
+description: |
+  In order for a screen reader to present a spoken UI to the user, meaningful
+  elements must have proper labels or text alternatives. A label or text
+  alternative gives an element its accessible name, one of the key properties
+  for expressing element semantics in the accessibility tree.
+web_lighthouse: 
+  - label
+  - image-alt
+  - input-image-alt
+  - frame-title
+  - object-alt
+  - document-title
+  - button-name
+  - link-name
+wf_blink_components: N/A
+---
+
+# Labels and text alternatives
+
+In order for a screen reader to present a spoken UI to the user, meaningful
+elements must have proper labels or text alternatives. A label or text
+alternative gives an element its accessible **name**, one of the key properties
+for [expressing element semantics in the accessibility
+tree](https://web.dev/accessible/semantics-and-screen-readers/#semantic-properties-and-the-accessibility-tree).
+
+When an element's name is combined with the element's **role**, it gives the
+user context so they can understand what type of element they're interacting
+with, and how it is represented on the page. If a name is not present, then a
+screen reader will only announce the element's role. Imagine trying to navigate
+a page and hearing, "button", "checkbox", "image" without any additional
+context. This is why labeling and text alternatives are crucial to a good,
+accessible experience.
+
+## Inspect an element's name
+It's easy to check an element's accessible name using Chrome's DevTools.
+
+1. Right-click on an element and choose **Inspect**. This will open the DevTools
+   Elements panel.
+1. In the Elements panel, look for the **Accessibility** pane. It may be hidden
+   behind a `»` symbol.
+1. In the **Computed Properties** dropdown, look for the **Name** property.
+
+![DevTools accessibility pane showing the computed name for a button.](./devtools-name.png)
+
+<div class="aside note">
+To learn more, check out the <a href="https://developers.google.com/web/tools/chrome-devtools/accessibility/reference">
+DevTools Accessibility Reference
+</a>.
+</div>
+
+Whether you're looking at an `img` with `alt` text, or an `input` with a
+`label`, you'll soon discover that all of these things result in the same
+outcome: giving an element its accessible name.
+
+## Check for missing names
+There are a number of accessibility and SEO audits in Lighthouse that will test
+for missing accessible names.
```

Comment:
Another 'will' that can change. I'm not gonna mark all of them-- but putting in a second comment just as a reminder to sweep for 'will'.